chromium/background.js

```diff
@@ -130,11 +130,13 @@ function updateState () {
     iconState = 'blocking';
   }
 
-  chrome.browserAction.setIcon({
-    path: {
-      38: 'icons/icon-' + iconState + '-38.png'
-    }
-  });
+  if ('setIcon' in chrome.browserAction) {
```

Comment:
`if (chrome.browserAction.setIcon) {` would work just as well and is more readable.

Reply:
I actually think this is more readable.

Reply:
It's a bit harder to read in my opinion. It's common in JS to use such property existence checks.

Reply:
Even if so, I like the explicit method you're searching for separated, as a syntactic nicety.